test/github-auth.js

```diff
@@ -55,6 +57,9 @@ class FakeGithub {
           throwError(404);
         }
       },
+      'integrations.getInstallationRepositories': async() => {
+        return this.repositories[installation_id];
+      },
```

Comment:
I saw your ping about this last night, but it looks like you figured it out.  This works great.

Two minor cleanups though:
 * `this.installation_id` is never used (instead, you use `installation_id` directly here, which is fine)
 * We make a new `FakeGithub` instance for each installation, so `this.repositories` will only ever have one property, the installation ID.  So you could simplify it a little by not indexing it with the installation ID:
```
      'integrations.getInstallationRepositories': async() => {
        return this.repositories;
      },
```
and
```

  setRepositories(repoName) {
    this.repositories = {repositories: [...repoName].map(repo => {name: repo})}; 
  }
```